content/en/apps/tutorials/local-setup.md

```diff
@@ -63,39 +86,38 @@ If you have trouble installing `medic-conf`, see the application's [GitHub repos
 
 *****
 
-### 2. Install the Core Framework
-
-Download the [docker-compose.yml](https://github.com/medic/cht-core/blob/master/docker-compose.yml) file to a folder of your choice. Be sure to place it in a folder that's easy to find on terminal or command line.
+### 3. Upload Test Data
 
-Open your terminal or command line and navigate to the folder where you have your [docker-compose.yml](https://github.com/medic/cht-core/blob/master/docker-compose.yml) file and run the command:
+By default, the CHT will have the [Maternal & Newborn Health Reference Application]({{< ref "apps/examples/anc" >}}) installed. To upload demo data you can use `medic-conf`:
 
-```zsh
-docker-compose up
-```
+{{< figure src="test.data.png" link="test.data.png" class="right col-3 col-lg-6" >}}
 
-{{< figure src="medic-login.png" link="medic-login.png" class="right col-6 col-lg-8" >}}
+- Navigate your terminal to the `cht-core/config/default` directory. This is where the reference application is stored.
+- Run the following `medic-conf` command to compile and upload default test data to your local instance: 
 
-Once the command is done running, navigate to `https://localhost` on a Google Chrome browser and login with the default username `medic` and default password `password`.
+```shell  
+medic-conf --url=https://medic:password@localhost --accept-self-signed-certs csv-to-docs upload-docs`.
+```
 
-If you encounter an error `bind: address already in use`, see the [Port Conflicts section]({{< relref "core/guides/docker-setup#port-conflicts" >}}) in our Docker Setup guide
+With the test data uploaded, log back into your CHT instance and note the "Test Health Facility" and related data.
 
 <br clear="all">
 
  *****
 
-### 3. Create and Upload a Blank Project
+### 4. Create and Upload a Blank Project
 
-With `medic-conf` you can create a blank project. This provides you a template from which you can begin working on CHT. Just run:
+With `medic-conf` you can also create a blank project. This provides you a template from which you can begin working on CHT. Just run:
```

Comment:
Might help to be a little clearer and formal on what we are asking them to run, but given that this is a tutorial, it is also probably ok to be less formal.

```suggestion
With `medic-conf` you can also create a blank project. This provides you a template from which you can begin working on CHT. To do so, run the following commands:
```